### DISCUSSION TITLE
Is it possible to separate Models by Team?

### DISCUSSION THREAD

**Discussion by abikali** [BerriAI/litellm#5563] [Q&A] [answered]
Hey,

I'm trying LiteLLM out, and so far what it's doing is brilliant! Currently using the Proxy Server.

Currently we're able to create Virtual Keys by selected team.
Is it possible to create Models by Team?

I'm asking that because I have multiple teams, but each will have his own OpenAI API key, so technically, in the current approach, I would have model for every team based on for example gpt-4. 

Technically, it would be awesome if there's a way to separate Models as we separate Virtual Keys, by selected team. 

|  Team  | Model | API_Key |
|--------|-------|---------|
| Team A | gpt-4 |    1111 |
| Team B | gpt-4 |    2222 |
| Team C | gpt-4 |    3333 |
| Team D | gpt-4 |    4444 |

**Comment by ishaan-jaff** [comment, marked as answer]
yes - use this https://docs.litellm.ai/docs/proxy/team_based_routing

**Reply by SAshree1313** [reply]
Is this a free tier service? Because right now even after using the teams my provider key used are random.
